src/CalendarApp/CalendarApp.css

```diff
@@ -1,80 +1,124 @@
-
-
-
 .calendarTitle {
-    padding: 20px;
+    padding-bottom: 10px;
     color: grey;
-    font-size: 20px;
+    font-size: 18px;
     margin: 0;
+    text-align: start;
```

Comment:
Make sure that is needed and please let me know as I have not seen this before.

Reply:
probs not needed now that you have changed it to text align left as default for the entire app.

that i think was more a typo that anything else...